plugins/protocols/session-lock.cpp

```diff
@@ -318,7 +318,7 @@ class wf_session_lock_plugin : public wf::plugin_interface_t
             if (state == ZOMBIE)
             {
                 output->set_inhibited(true);
-                output_states[output]->crashed_node->display();
+                output_states[output]->crashed_node->display("💥");
```

Comment:
Maybe we can define this emoji somewhere as a constant in order to not repeat it? :)

Reply:
Yes, makes sense -- I've added a separate function for it